chart/openfaas/templates/oidc-plugin-rbac.yaml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
I think this needs an "if statement" for if the plugin is enabled?

Reply:
This has been updated.